packages/react/src/create-strict-api/__tests__/index.test.tsx

```diff
@@ -79,7 +86,7 @@ describe('createStrictAPI()', () => {
         color: 'var(--ds-text)',
         all: 'inherit',
         '&:hover': { color: 'var(--ds-text-hover)' },
-        '&:invalid': { color: 'orange' },
+        '&:invalid': { color: 'var(--ds-text)' },
```

Comment:
All psuedos are now enforced to use the "base" values from the schema. This fixes the type error.